app/models/miq_widget.rb

```diff
@@ -308,6 +308,8 @@ def generate_report_result(rpt, owner, timezone = nil)
     name = owner.respond_to?(:userid) ? owner.userid : owner.name
     group = owner.kind_of?(MiqGroup) ? owner : owner.try(:current_group)
 
+    raise _("Unable to generate report result for owner %{name} because owner group is nil") % {:name => name} if group.nil?
```

Comment:
Heh, I was going to ask "why aren't you using string interpolation for this", until I realized that wouldn't work with a string translation, so this really is what you need to do. 👍 

(also, clearly I don't deal with translated strings enough, because I am sure this would have been obvious to others)